pageserver/src/object_repository.rs

```diff
@@ -860,105 +878,29 @@ impl<'a> History for ObjectHistory<'a> {
     }
 }
 
-impl<'a> ObjectHistory<'a> {
-    fn handle_relation_size(
-        &mut self,
-        rel_tag: RelTag,
-        entry: RelationSizeEntry,
-    ) -> Option<Update> {
-        match entry {
-            RelationSizeEntry::Size(size) => {
-                // we only want to output truncations, expansions are filtered out
```

Comment:
Since we're only putting raw data into the object store, I guess we can't filter out expansions anymore?

Reply:
Why do we need to filter them? We just copy all versions bother of data and metdata.